cmd/simRecon/simRecon.go

```diff
@@ -52,32 +52,52 @@ func ReconstructSeq(newickInput string, fastaInput string, outputFilename string
 }
 
 //SimRecon simulates evolution, performs reconstruction, and then evaluates the accuracy of the reconstruction
-func SimRecon(rootFastaFile string, treeFile string, gp string, simOutFile string, leafOutFile string, reconOutFile string, accuracyOutFile string) {
+func SimRecon(rootFastaFile string, treeFile string, gp string, simOutFile string, leafOutFile string, reconOutFile string, accuracyOutFile string, option int, baseAccFile string) {
```

Comment:
To make sure I'm reading this correctly, there is no "output" other than what's printed to the screen?

Reply:
What function would output this "baseAccFile"? Is this the reconstruct.ReconAccuracy output "accBases"? Or maybe you can tell me more about how this file is made?

Reply:
How do you know the location that the new file will be written into? Is it based on the pwd where you execute the function?